code/go/0chain.net/blobbercore/allocation/file_changer_update.go

```diff
@@ -81,9 +76,6 @@ func (nf *UpdateFileChanger) ApplyChange(ctx context.Context, change *Allocation
 	fileRef.HashToBeComputed = true
 	nf.deleteHash = make(map[string]int)
 
-	if fileRef.ThumbnailHash != "" && fileRef.ThumbnailHash != nf.ThumbnailHash {
```

Comment:
Are we removing thumbnail at all? If so its fine.

Reply:
No we are not, but we don't increase the number of files or file size when thumbnail is uploaded so we don't delete it here as well. We delete this from filestore on next commit.